<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teknikServisTable</class>
 <widget class="QDialog" name="teknikServisTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teknik Servis Tablosu</string>
  </property>
  <widget class="QTableWidget" name="tbl_teknikservis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_anamenu">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>AnaMenü</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ara">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>İsim Giriniz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ara">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ara</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
